Signal/src/ViewControllers/SelectThreadViewController.m

```diff
@@ -131,11 +132,13 @@ - (void)updateTableContents
     __weak SelectThreadViewController *weakSelf = self;
     ContactsViewHelper *helper = self.contactsViewHelper;
     OWSTableContents *contents = [OWSTableContents new];
-    OWSTableSection *section = [OWSTableSection new];
 
-    // Threads
+    // Existing threads are listed first, ordered by most recently active
+    OWSTableSection *recentChatsSection = [OWSTableSection new];
+    recentChatsSection.headerTitle = NSLocalizedString(
+        @"SELECT_THREAD_TABLE_RECENT_CHATS_TITLE", @"Table section header for recently active conversations");
```

Comment:
Just spitballing here: I wonder if we should interleave these search results (visually distinguishing contacts from conversations somehow) and sort them by "match weight" where, for example, matches at the start of a name would be weighted more heavily.

e.g. searching for "Bo", a user would see "Bob James <contact>" before "Trip to Boulder <group>" and a conversation with "Bob James <1:1>" before "Alice Arecibo <contact>".

Reply:
Happy to discuss this more for a future iteration.